images/access-setup/content/bin/setup_compute.sh

```diff
@@ -176,6 +197,26 @@ tekton_chains_manifest(){
   printf "OK\n"
 }
 
+tekton_results_manifest(){
+  manifest="$manifests_dir/compute/tekton-results/tekton-results-secret.yaml"
+  if [ ! -e "$manifest" ]; then
+    manifests_tmp_dir="$(dirname "$manifest")/tmp"
+    mkdir -p "$manifests_tmp_dir"
+    {
```

Comment:
The TEKTON_RESULTS_* vars should probably be checked before generating the secret, and an error returned if unset.

Reply:
added check and error output